### PR TITLE
Support for adding GCE internal TCP load balancer

### PR DESCRIPTION
**Pull request by liorr**
Added support for creating an GCE internal load balancer to the ES cluster group.

### CODE REVIEW

File: src/vendor_specific/gce/gce.env.sample [hunk 1]
```diff
@@ -11,3 +11,4 @@ SCOPES=default,storage-full,compute-rw
 SUBNET=
 ENTERPRISE_ENABLED=true
 DISK_TYPE=pd-ssd
+EXTERNAL_IP=false
```
Comment (romankor) [inline src/vendor_specific/gce/gce.env.sample:14]: I think that should be true by default , there is no way to debug your startup sequence without the external IP , also external IP is not so bad , if not open in all the ports

Comment (liorr) [reply src/vendor_specific/gce/gce.env.sample:14]: I think as a default more secure is better

File: src/vendor_specific/gce/gce.env.sample [hunk 2]
```diff
@@ -11,3 +11,6 @@ SCOPES=default,storage-full,compute-rw
...
+LB_NETWORK=elasticsearch
```
Comment (romankor) [inline src/vendor_specific/gce/gce.env.sample:15]: if empty take from `NETWORK` same for `SUBNETWORK`

**Review by romankor — APPROVED** [review]

File: src/vendor_specific/gce/create-lb.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+if [ -z "$LB_NETWORK" ]
+then
+    LB_NETWORK=$NETWORK
+fi
+
+if [ -z "$LB_SUBNET" ]
+then
+    LB_SUBNET=$SUBNET
+fu
```
Comment (romankor) [inline src/vendor_specific/gce/create-lb.sh]: typo `fi`

**Review by romankor — CHANGES_REQUESTED** [review]

**Review by romankor — APPROVED** [review]

File: src/vendor_specific/gce/create-lb.sh (hunk 3 above)
Comment (minhtule) [reply src/vendor_specific/gce/create-lb.sh]: Just see this while drinking water... almost choke 😂

**Pull request merged**